apps/server/cypress/e2e/suite1/testExample.cy.ts

```diff
@@ -38,6 +40,55 @@ const makeABConnection = async (jobType) => {
   expectConnectionSuccess();
 };
 
+const getAccountId = ({ aggregator, memberGuid, userId }) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/connection/${memberGuid}/accounts`;
+
+  return cy.request("get", `/api${url}`).then((dataResponse) => {
+    const accountId = dataResponse.body.accounts.find(
+      (acc) => Object.keys(acc)[0] === "depositAccount",
+    ).depositAccount.accountId;
+
+    return accountId;
+  });
+};
+
+const verifyTransactionsValidatorSuccess = ({
+  accountId,
+  aggregator,
+  userId,
+  queryString = "",
```

Comment:
Why do we need an aggregator or queryString variable if it's always the same?

Reply:
Good point. Removed

Reply:
Removing aggregator....

Reply:
Done